### PR TITLE
refactor: update intro and welcome pages

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Set the correct content for the intro and welcome pages.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-832/change-intro-meow-world-quick-start
https://linear.app/autokitteh/issue/UI-836/update-intro-and-welcome-pages-content

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [x] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/dashboard/welcomeMainBlock.tsx [hunk 1]
```diff
@@ -104,27 +102,28 @@ export const DashboardWelcomeMainBlock = () => {
...
 				</WelcomeInfoCard>
 
 				<WelcomeInfoCard
-					onPlay={() => handleOpenModal("https://www.youtube.com/embed/QWSa0etwTDE")}
+					onPlay={() => handleOpenModal("https://www.youtube.com/embed/zNtJ8OBPUmY")}
 					title={
 						<Typography className="text-xl font-bold" element="h3">
-							{t("cards.developInVSCode.developInVSCode")}{" "}
-							<Link className="font-normal text-green-800" to="#">
+							<div className="inline">{t("cards.developInVSCode.developInVSCode")}</div>{" "}
```
Comment (J1za) [inline src/components/organisms/dashboard/welcomeMainBlock.tsx:123]:
```suggestion
							{t("cards.developInVSCode.developInVSCode")}{" "}
```

File: src/components/organisms/dashboard/welcomeMainBlock.tsx [hunk 2]
```diff
@@ -139,6 +138,20 @@ export const DashboardWelcomeMainBlock = () => {
...
+					<Link
+						className="flex flex-row items-center hover:text-green-200"
+						target="_blank"
+						to="https://discord.gg/UhnJuBarZQ"
+					>
+						<IconSvg className="ml-2 mr-1" size="lg" src={DiscordIcon} />{" "}
+						<div className="underline">{t("cards.footer.joinDiscord")}</div>
+					</Link>
+				</Typography>
+			</div>
```
Comment (J1za) [inline src/components/organisms/dashboard/welcomeMainBlock.tsx:154]:
```suggestion
			<div className="rounded-xl border border-gray-950 bg-gray-1250 py-5 pl-6 pr-4 font-averta">
				<Typography className="flex h-full items-center justify-center text-lg" element="p">
					{t("cards.footer.haveAQuestion")}
					<Link
						className="flex items-center gap-0.5 hover:text-green-200"
						target="_blank"
						to="https://discord.gg/UhnJuBarZQ"
					>
						<IconSvg className="ml-2 mr-1" size="lg" src={DiscordIcon} />
						<div className="underline">{t("cards.footer.joinDiscord")}</div>
					</Link>
				</Typography>
			</div>
```

File: src/components/organisms/dashboard/welcomeMainBlock.tsx [hunk 3]
```diff
@@ -104,27 +102,28 @@ export const DashboardWelcomeMainBlock = () => {
...
 
 				<WelcomeInfoCard
-					onPlay={() => handleOpenModal("https://www.youtube.com/embed/QWSa0etwTDE")}
+					onPlay={() => handleOpenModal("https://www.youtube.com/embed/zNtJ8OBPUmY")}
 					title={
 						<Typography className="text-xl font-bold" element="h3">
-							{t("cards.developInVSCode.developInVSCode")}{" "}
-							<Link className="font-normal text-green-800" to="#">
+							<div className="inline">{t("cards.developInVSCode.developInVSCode")}</div>{" "}
+							<div className="inline text-green-800">
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/dashboard/welcomeMainBlock.tsx]: refactored

File: src/components/organisms/dashboard/welcomeMainBlock.tsx (hunk 2 above)
Comment (RonenMars) [reply src/components/organisms/dashboard/welcomeMainBlock.tsx]: done

**Review by J1za — APPROVED** [review]

**Pull request merged**